routers/user/setting/profile.go

```diff
@@ -32,6 +32,7 @@ const (
 func Profile(ctx *context.Context) {
 	ctx.Data["Title"] = ctx.Tr("settings")
 	ctx.Data["PageIsSettingsProfile"] = true
+
```

Comment:
Why this change?

Reply:
Thought it is more consistent with the rest of the code base.
But of course I can remove it :)

Reply:
@JonasFranzDEV Do you want me to remove it?